plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -1120,4 +1120,34 @@ SecRule REQUEST_FILENAME "@endsWith /wp-admin/admin-ajax.php" \
             "t:none,\
             ctl:ruleRemoveTargetById=942360;ARGS:s"
 
+# Uploading plugins as a zip file
+SecRule REQUEST_FILENAME "@endsWith /wp-admin/update.php" \
+    "id:9507973,\
+    phase:2,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'wordpress-rule-exclusions-plugin/1.0.1',\
+    chain"
+    SecRule ARGS:action "@streq upload-plugin" \
+        "t:none,\
+        chain"
+        SecRule FILES:pluginzip "@endsWith .zip" \
+            "t:none,\
+            ctl:ruleRemoveTargetById=944110;REQUEST_BODY,\
+            ctl:ruleRemoveTargetById=944250;REQUEST_BODY"
+
+# Installing a plugin via zip file upload
+SecRule REQUEST_FILENAME "@endsWith /wp-admin/update.php" \
+    "id:9507974,\
+    phase:2,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'wordpress-rule-exclusions-plugin/1.0.1',\
+    chain"
+    SecRule ARGS:install-plugin-submit "@streq Install Now" \
```

Comment:
I think we can remove this condition.

Reply:
It seems a bit overkill since the false positive is exactly with ``Install Now``.

Reply:
It is also an exclusion for submit button caption which si probably not used at all by the application.

Reply:
true...